screenshot-testing/src/main/java/com/stripe/android/screenshottesting/PaparazziRule.kt

```diff
@@ -19,14 +23,13 @@ import org.junit.runners.model.Statement
 
 class PaparazziRule(
     vararg configOptions: List<PaparazziConfigOption>,
-    private val boxModifier: Modifier = Modifier,
+    private val boxModifier: Modifier = Modifier.defaultBoxModifier(),
```

Comment:
I think this will fail the `Link` screenshots as a default box modifier was not added to its test rules. If you have to regenerate, let me know.

Reply:
Or you could specify the `Modifier` (without boxModifier) when creating the rule in the link module tests.

Reply:
All the link tests are passing locally for me, should I update anyways?

Reply:
No need to update if they are passing.

Reply:
Updated as they were failing on CI